functions/generic/lambda_function.py

```diff
@@ -0,0 +1,40 @@
+from socless import socless_bootstrap
+from slack_sdk.web.client import WebClient
+from slack_sdk.web.internal_utils import _parse_web_class_objects
+
+from slack_helpers import SOCLESS_BOT_TOKEN
+
+
+def handle_state(
+    api_method: str,
+    http_verb: str = "POST",
+    params: dict = None,  # type: ignore
+    json: dict = None,  # type: ignore
+    token: str = "",
+):
+    """Create slack channel and invite any provided slack_ids.
```

Comment:
Doc string here needs updating

Reply:
fixed the docstring and added an example of usage